crates/bevy_ui/src/lib.rs

```diff
@@ -173,13 +174,18 @@ impl Plugin for UiPlugin {
 
             system
         });
-        app.add_systems(
-            PostUpdate,
-            widget::update_atlas_content_size_system.before(UiSystem::Layout),
-        );
+
         app.add_systems(
             PostUpdate,
             (
+                (
+                    widget::update_atlas_content_size_system,
+                    update_target_camera_system,
+                )
+                    .before(UiSystem::Layout),
+                apply_deferred
```

Comment:
bit of a shame to `apply_deferred` here.

could we instead add the `TargetCamera` to `NodeBundle` (and variants) and then mutably update it in `update_target_camera_system`, which would avoid needing to apply commands before layouting?

Reply:
This `apply_deferred` also smells to me a bit, but I am not super familiar with the technique you are referring to and would ask for an example and probably another opinion here.

Reply:
This is not blocking, I’ll make a pr for it after merge. I appreciate the effort so far and the dwindling motivation, will re-review and approve shortly so we can get it over the line.